src/darwin/agent/launchd.js

```diff
@@ -253,6 +253,10 @@ function findJbdCallImpl() {
   return matches[0].address;
 }
 
+function stringToHexPattern(str) {
+  return str.split('').map(o => o.charCodeAt(0).toString(16)).join(' ');
+}
```

Comment:
Let's move this function below `findSubstrateLauncher()`. (Newspaper code structure.)